global.js

```diff
@@ -161,7 +161,22 @@ if (!ts.eventsRegistered) {
 
     // Events.on(EventType.PlayerJoin, cons(e => teamKeeper(e.player, false)))
     // Events.on(EventType.PlayerLeave, cons(e => teamKeeper(e.player, true)))
-
+    
+    var lastTick = 0
+    var mean = new WindowedMean(60)
+    const timer = new Interval()
+    Events.run(Trigger.update, () => {
+        mean.add(Time.timeSinceMillis(lastTick))
+        lastTick = Time.millis()
+        if (timer.get(60)) { // Run once a second
+            const tps = 1000 / mean.rawMean()
+            Groups.player.each(cons(p => {
+                Call.infoPopup(p.con, `TPS: ${tps}`, 1,  Align.top, p.con.mobile ? 0 : 120, 0, 0, 0) // 3 times to approximately match the background color of the item hud
+                Call.infoPopup(p.con, `TPS: ${tps}`, 1,  Align.top, p.con.mobile ? 0 : 120, 0, 0, 0)
+                Call.infoPopup(p.con, `TPS: ${tps}`, 1,  Align.top, p.con.mobile ? 0 : 120, 0, 0, 0)
```

Comment:
```suggestion
                Call.infoPopup(p.con, 'TPS: ' + tps, 1,  Align.top, p.con.mobile ? 0 : 120, 0, 0, 0) // 3 times to approximately match the background color of the item hud
                Call.infoPopup(p.con, 'TPS: ' + tps, 1,  Align.top, p.con.mobile ? 0 : 120, 0, 0, 0)
                Call.infoPopup(p.con, 'TPS: ' + tps, 1,  Align.top, p.con.mobile ? 0 : 120, 0, 0, 0)
```
rhino doesnt support those
![image](https://user-images.githubusercontent.com/48705529/120102678-97594600-c165-11eb-9f26-5dc5df8e2c5d.png)